locations/spiders/raleys_us.py

```diff
@@ -1,40 +1,47 @@
-from scrapy.http import JsonRequest
+from scrapy.http import JsonRequest, Request
 
-from locations.categories import Categories
+from locations.categories import Categories, apply_category
 from locations.hours import OpeningHours
 from locations.json_blob_spider import JSONBlobSpider
 
-BRAND_WIKIDATA = {
-    "Bel Air": "Q112922067",
-    "Nob Hill Foods": "Q121816894",
-    "Raley's": "Q7286970",
-    "Raley's ONE Market": "Q7286970",
+BRAND_MAP = {
+    "Bel Air": {"brand": "Bel Air", "brand_wikidata": "Q112922067"},
+    "Nob Hill Foods": {"name": "Nob Hill Foods", "brand": "Nob Hill Foods", "brand_wikidata": "Q121816894"},
```

Comment:
Did you create [Q121816894](https://www.wikidata.org/wiki/Q121816894)? It has been deleted

Reply:
There used to be a Wikipedia article for Nob Hill Foods, and that was the QID, but the article got deleted for being not notable.